core/kernel/show_conf.c

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 1019 Huawei Technologies Co., Ltd
+ */
+
+#include <initcall.h>
+#include <trace.h>
+
+extern const char conf_str[];
+
+static TEE_Result show_conf(void)
+{
+#if (TRACE_LEVEL >= TRACE_INFO)
+	IMSG("Contents of conf.mk (decode with 'ascii85 -d | xz -d'):");
```

Comment:
Besides a bit smaller size in terms of output, why not go with base64 instead that seems to be way more widespread? Or just simply post it as plaintext?

Reply:
base64 is only ~7% larger so yes maybe better.

```
$ cat optee_os/out/arm/conf.mk | xz | ascii85 | wc
     23      24    1883
$ cat optee_os/out/arm/conf.mk | xz | base64 | wc
     27      27    2011
```